packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -317,13 +323,28 @@ class _DraggableScrollableSheetState extends State<DraggableScrollableSheet> {
       }
       _extent._currentExtent.value = _extent.initialExtent;
     }
+    _child = widget.builder(context, _scrollController);
+  }
+
+  @override
+  void didUpdateWidget(DraggableScrollableSheet oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    if (oldWidget.builder != widget.builder) {
+      _child = widget.builder(context, _scrollController);
+    }
+
+    _extent = _DraggableSheetExtent(
```

Comment:
This is cheap to create, so it seems silly to check the min/max/initial sizes for changes.

Reply:
Is this still true even with us now creating a new _scrollController as well, which will cause other widgets to move their listeners over to the new one?

Reply:
I think it's probably still fine